src/Microsoft.Azure.WebJobs.Extensions.Sql.csproj

```diff
@@ -1,17 +1,21 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-
   <PropertyGroup>
     <TargetFramework>netstandard2.1</TargetFramework>
     <Description>SQL binding extension for Azure Functions</Description>
     <Company>Microsoft</Company>
     <Authors>Microsoft</Authors>
     <Product>SQL Binding Extension</Product>
+    <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
     <GenerateAssemblyInfo>true</GenerateAssemblyInfo>
     <DebugSymbols>true</DebugSymbols>
     <IncludeSymbols>false</IncludeSymbols>
     <DebugType>embedded</DebugType>
     <PackageId>Microsoft.Azure.WebJobs.Extensions.Sql</PackageId>
+    <PackageTags>Microsoft Azure WebJobs AzureFunctions SQL</PackageTags>
```

Comment:
Is there an "AzureSQL" package tag that we should add as well?

Reply:
I'm not seeing anything like that when I search https://www.nuget.org/packages?q=azure+sql

Is AzureSQL (all one word) something you think people would actually be searching? As long as you have azure or sql in it then it'll be fine and that seems to me to be the more common phrases to search

Reply:
I have no data to support "AzureSQL". Just wondering if there was another tag beyond "SQL", that's all 😄 . Totally fine.

Reply:
Added it, not like it's going to cause problems and might be useful to someone